engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/semantic/TypeSignaturesTest.scala

```diff
@@ -202,41 +239,41 @@ class TypeSignaturesTest
           |foo a = 42""".stripMargin
       val module = code.preprocessModule
       getSignature(module, "foo") should typeAs(
-        "Unnamed.Test.A" ->: "Unnamed.Test.B" ->: "Unnamed.Test.C" ->: "Unnamed.Test.C" ->: "Unnamed.Test.A"
+        "local.My_Lib.Test.A" ->: "local.My_Lib.Test.B" ->: "local.My_Lib.Test.C" ->: "local.My_Lib.Test.C" ->: "local.My_Lib.Test.A"
       )
     }
 
-    "resolve imported names" in {
+    "XX resolve imported names" in {
```

Comment:
Why is this prefix added?

Reply:
Sorry, this is a leftover. I usually add "XX" prefix to a specific scala test so that I can run/debug it easily with a command like `test *TypeSignaturesTest -- -z XX`.

Reverted in another PR in https://github.com/enso-org/enso/pull/11468/commits/845e03040864dc25377c80b3e6bb121654298133